docs/process/retreats.md

```diff
@@ -126,3 +126,15 @@ Remember, throughtout agenda planning, the previous planners and the co-founders
 **The week following the retreat, we aim to have the following completed:**
 * The decision as to which Flexpal/s will be leading the next retreat
 * Feedback on each Flexpal has been shared around how they showed up to the retreat
+
+---
+
+## Travel
+
+Be sure to book your travel as soon as possible. We recommend booking flights at least 6 weeks in advance to get the best rates.
+
+When booking your flights, please keep the following in mind:
+
+* **Arrival**: Please plan to arrive on Monday evening before the retreat starts on Tuesday morning.
+* **Departure**: Please plan to depart after Thursday evening when the retreat ends.
```

Comment:
Maybe add, "many Flexpals leave Friday AM but sometimes a Thursday PM flight is a necessity and the team understands." or something to that effect?